docs/source/locale/fr/LC_MESSAGES/arch/data_catalog.po

```diff
@@ -53,4 +53,3 @@ msgid ""
 " will search for all files that are part of the RCP8.5 experiment and "
 "based on the CRCM4 model."
 msgstr ""
-
```

Comment:
All the `.po` files seem to have an empty last line.  Is this an accident or the format requires it?

Reply:
It's a setting for one of the hooks. I should be able to relax it for `.po` files.

Reply:
CRCM4 n'a jamais été piloté avec des données RCP8.5
Nos vieilles simulations CRCM$ ont été forcées avec des GCM de CMIP3 forcées par SRES A2